### PR TITLE
Eng 128: Add font family

### PR DESCRIPTION
**Pull request by afolasope**

### CODE REVIEW

**Review by AfeezGL — CHANGES_REQUESTED** [review]

File: apps/frontend/tailwind.config.ts [hunk 1]
```diff
@@ -4,6 +4,9 @@ const config: Config = {
     content: ["./src/**/*.{html,js,jsx,ts,tsx,mdx}", "../../node_modules/@tremor/**/*.{js,ts,jsx,tsx}"],
     theme: {
         extend: {
+            fontFamily: {
+                inter: ["Inter"],
```
Comment (AfeezGL) [inline apps/frontend/tailwind.config.ts:8]:
```suggestion
                sans: ["Inter"],
```

@afolasope to customize the default font in tailwindcss, we update the `theme.fontFamily.sans` in our tailwind config file or add a custom `base` style that sets `font-family` in our `css` file e.g.
```css
@layer base {
  html {
    font-family: "Inter", system-ui, sans-serif;
  }
}
```

[more...](https://tailwindcss.com/docs/font-family#customizing-the-default-font)

File: apps/frontend/src/assets/fonts/Inter/Inter-VariableFont_slnt,wght.ttf
Comment (jdamilola) [inline apps/frontend/src/assets/fonts/Inter/Inter-VariableFont_slnt,wght.ttf:1]: @afolasope Can we get this font from a reliable CDN instead? Bundling will increase our build by 786KB.

**Review by jdamilola — CHANGES_REQUESTED** [review]
Can you address my comment?

File: apps/frontend/index.html [hunk 2]
```diff
@@ -3,6 +3,7 @@
     <head>
         <meta charset="UTF-8" />
         <link rel="icon" type="image/x-icon" href="/favicon.ico" />
+        <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
```
Comment (jdamilola) [inline apps/frontend/index.html:6]: @afolasope It looks like this is from a personal website and not a CDN, so it reliability is not guaranteed. Can you use https://fonts.googleapis.com/css?family=Inter instead?

Comment (afolasope) [reply apps/frontend/index.html]: > @afolasope It looks like this is from a personal website and not a CDN, so it reliability is not guaranteed. Can you use https://fonts.googleapis.com/css?family=Inter instead?

I found it in the tailwind documentation but i have changed it now

**Review by AfeezGL — APPROVED** [review]

**Review by jdamilola — APPROVED** [review]

**Pull request merged**